### PR TITLE
Composite volume tests load and attach

### PR DESCRIPTION
**Pull request by pdebski21**

### CODE REVIEW

**Review by arutk — CHANGES_REQUESTED** [review]

**Comment by bkarpins** [comment]
valgrind this please

**Comment by bkarpins** [comment]
valgrind this please

**Comment by arutk** [comment]
retest this please

File: tests/functional/tests/management/test_composite_volume.py [hunk 1]
```diff
@@ -363,4 +385,58 @@ def test_load(pyocf_ctx):
...
+    io.callback = cmpl.callback
+    io.submit()
+    cmpl.wait()
+    assert 0 == int(cmpl.results["err"])
+
+    cache.stop()
+
+    cvol = CVolume(pyocf_ctx)
+    for v in vols:
+        cvol.add(v)
```
Comment (jfckm) [inline tests/functional/tests/management/test_composite_volume.py:417]: Is this necessary to create this composite volume again? Can't the test just use `cvol` again?

**Review by jfckm — CHANGES_REQUESTED** [review]

File: tests/functional/tests/management/test_composite_volume.py (hunk 1 above)
Comment (pdebski21) [reply tests/functional/tests/management/test_composite_volume.py:417]: Yes it was an issue. Without second initialization segfault occurs.
During cache.stop() there is some deinitialization of cvolume.

**Review by jfckm — APPROVED** [review]

**Review by jfckm — APPROVED** [review]

**Review by mmichal10 — APPROVED** [review]

**Pull request merged**